src/components/views/lobby/Lobby.tsx

```diff
@@ -18,6 +18,7 @@ import "../../../styles/ui/images.scss"
 import { Game } from "../../../generated"
 
 const Lobby = observer(() => {
+  const MAX_NUMBER_OF_PLAYERS = 5
```

Comment:
ich persönlich würd das nicht so schreiben. Bis jetzt hatten wir für Frontend sachen überall camel case "playerSubscription, startPollPlayers, etc" - ich schlage vor, du schreibsts einfach maxNumberPlayers oder so, damit es einfach schöner aussieht

TRANSLATION FOR MEHLI: I want beat to rename this constant so that it's in camel case and not Capslock with the underlines (I suggested maxNumberPlayers)

Reply:
Passt, ich hab's from backend kopiert. Da hast du absolut recht.